static/js/components/PrivacyForm.js

```diff
@@ -0,0 +1,64 @@
+/* global SETTINGS: false */
+import React from 'react';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+
+import ProfileProgressControls from './ProfileProgressControls';
+import ProfileFormFields from '../util/ProfileFormFields';
+import {
+  combineValidators,
+  personalValidation,
+  educationValidation,
+  educationUiValidation,
+  employmentValidation,
+  employmentUiValidation,
+  privacyValidation,
+} from '../util/validation';
+
+class PrivacyForm extends ProfileFormFields {
+  static propTypes = {
+    profile:        React.PropTypes.object,
+    saveProfile:    React.PropTypes.func,
+    updateProfile:  React.PropTypes.func,
+    ui:             React.PropTypes.object,
+    params:         React.PropTypes.object,
+  };
+
+  render() {
+    const { saveProfile, profile, ui, params: { username } } = this.props;
+    let _username = username || SETTINGS.username;
+
+    let profileProgressControlsOptions = {
+      nextUrl: "/dashboard",
+      isLastTab: true,
+      saveProfile: saveProfile,
+      profile: profile,
+      ui: ui,
+      validator: combineValidators(
+        personalValidation,
+        educationValidation,
+        educationUiValidation,
+        employmentValidation,
+        employmentUiValidation,
+        privacyValidation
+      )
+    };
+
+    if (this.props.location.pathname !== `/settings/${_username}` ) {
+      profileProgressControlsOptions['prevUrl'] = "/profile/professional";
+    }
+
+    return (
+      <Grid className="profile-tab-grid">
+        <Cell col={12}>
+          <h4>Who can see your profile?</h4>
+          { this.boundRadioGroupField(['account_privacy'], '', this.privacyOptions) } <br />
+        </Cell>
+        <Cell col={12}>
+          <ProfileProgressControls { ...profileProgressControlsOptions } />
```

Comment:
This shouldn't be part of the `PrivacyForm` since it's specific to the privacy tab. Can you move it to `PrivacyTab` similar to `EmploymentTab` and `EducationTab`?